<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createTemplate</class>
 <widget class="QDialog" name="createTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(242, 238, 203)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Създай шаблон&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Име на шаблона:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Основни</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxDatePublished">
       <property name="text">
        <string>Дата на публикуване</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBoxPages">
       <property name="text">
        <string>Брой станици</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxFiction">
       <property name="text">
        <string>Художествена / нехудожествена</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxName">
       <property name="text">
        <string>Име</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBoxSource">
       <property name="text">
        <string>Придобита от</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxPublisher">
       <property name="text">
        <string>Издателство</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBoxGenre">
       <property name="text">
        <string>Жанр</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxAuthor">
       <property name="text">
        <string>Автор</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="checkBoxFormat">
       <property name="text">
        <string>Формат</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBoxDatePublishedBulgaria">
       <property name="text">
        <string>Дата на публикуване в България</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBoxSeries">
       <property name="text">
        <string>Поредица</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxDateStarted">
       <property name="text">
        <string>Дата започната</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxDateFinished">
       <property name="text">
        <string>Дата завършена</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBoxScore">
       <property name="text">
        <string>Оценка</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="checkBoxNotes">
       <property name="text">
        <string>Общи бележки</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBoxCountry">
       <property name="text">
        <string>Държава на първо издание</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBoxLanguage">
       <property name="text">
        <string>Оригинален език</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBoxTranslation">
       <property name="text">
        <string>Преводна</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Романтика</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxSpicy">
       <property name="text">
        <string>Пикантност</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxTropes">
       <property name="text">
        <string>Тропи</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Фентъзи</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxFantasyKind">
       <property name="text">
        <string>Вид фентъзи</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxMagicSystem">
       <property name="text">
        <string>Магическа система</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Научна фантастика</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxScifiKind">
       <property name="text">
        <string>Вид фантастика</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxFuture">
       <property name="text">
        <string>Бъдеще</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Мистерия</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMystery">
     <property name="text">
      <string>Мистерия</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Хорър</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxScariness">
       <property name="text">
        <string>Страшна</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxDisgust">
       <property name="text">
        <string>Противност</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Исторически романи</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>Епоха</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>Историческа коректност</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Запиши</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
